modules/dicom_archive/php/viewdetails.class.inc

```diff
@@ -312,5 +312,18 @@ class ViewDetails extends \NDB_Form
             )
         );
     }
+
+    /**
+     * Generate a breadcrumb trail for that page.
+     *
+     * @return \LORIS\BreadcrumbTrail The trail.
+     */
+    public function getBreadcrumbs(): \LORIS\BreadcrumbTrail
+    {
+        $label = ucwords(str_replace('_', ' ', $this->name));
+        return new \LORIS\BreadcrumbTrail(
+            new \LORIS\Breadcrumb($label, "/$this->name"),
+            new \LORIS\Breadcrumb('View Details', $_SERVER['REQUEST_URI'])
```

Comment:
the reference to a superglobal makes me a little sad, but not as NDB_Breadcrumbs did.. could this feasibly just use the empty string or "#" as the link parameter instead?

Reply:
I will try to build the url base on the page